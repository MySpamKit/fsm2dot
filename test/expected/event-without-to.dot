digraph G {
  node [fontname = "Geneva"]
  edge [fontname = "Geneva"]
  splines=polyline
  one [shape="none", label=<<table style="rounded"><tr><td border="0">one</td></tr><hr/><tr><td border="0"></td></tr><hr/><tr><td border="0" align="left">do/ev</td></tr></table>>]
}