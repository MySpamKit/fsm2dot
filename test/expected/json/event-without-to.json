{"events":[],"choices":[],"states":[{"name":"one","activities":{"do":["ev"],"entry":[],"entered":[],"exit":[]}}],"final":[]}